iOS/New/Views/Reader/ReaderSettingsView.swift

```diff
@@ -204,6 +204,14 @@ struct ReaderSettingsView: View {
                                 ))
                             )
                         )
+                        SettingView(
+                            setting: .init(
+                                key: "Reader.pagedIsolateFirstPage",
+                                title: NSLocalizedString("ISOLATE_FIRST_PAGE"),
+                                notification: .init("Reader.pagedIsolateFirstPage"),
+                                value: .toggle(.init())
+                            )
```

Comment:
this setting should also be added to the `Settings.readerSettings` static variable to show in the settings tab